test/quantum_mechanics/harmonic_oscillator/energy_levels_test.py

```diff
@@ -0,0 +1,42 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (
+    assert_equal,
+    units,
+    Quantity,
+    errors,
+    prefixes,
+)
+from symplyphysics.laws.quantum_mechanics.harmonic_oscillator import energy_levels as energy_law
+
+# Description
+## The energy of the third level of a quantum harmonic oscillator for an angular frequency of
+## 3.14 rad/fs is E_3 = 7.23 eV.
+
+Args = namedtuple("Args", "n w")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    n = 3
+    w = 3.14 * units.radian / (prefixes.femto * units.second)
```

Comment:
I think we were making it `Quantity` in similar test.

Reply:
Sure enough